src/pages/landing/LandingContainer.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import Icon from "@assets/icons/Icon";
+import SignIn from "@components/landing/signIn/SignIn";
```

Comment:
Did you think to name the file just like `index`? In this case it would be better to avoid `signIn/SignIn`. If you replace SignIn for index, in the import it will be `@components/landing/signIn`

Reply:
I have fixed it in all the imports, it looks cleaner.